examples/literal/literal.cpp

```diff
@@ -0,0 +1,73 @@
+#include <biscuit/assert.hpp>
+#include <biscuit/assembler.hpp>
+
+#include <iostream>
+
+using namespace biscuit;
+
+constexpr const static uint64_t literal1_value = 0x1234567890ABCDEF;
```

Comment:
const and static can be removed here. constexpr already enforces const, and const file-scope constants have internal linkage by default.